tests/partial_model_tests.rs

```diff
@@ -27,32 +33,265 @@ mod entity {
 #[derive(FromQueryResult, DerivePartialModel)]
 #[sea_orm(entity = "Entity")]
 struct SimpleTest {
-    _foo: i32,
-    _bar: String,
+    foo: i32,
+    bar: String,
 }
 
 #[derive(FromQueryResult, DerivePartialModel)]
 #[sea_orm(entity = "<entity::Model as ModelTrait>::Entity")]
 struct EntityNameNotAIdent {
     #[sea_orm(from_col = "foo2")]
-    _foo: i32,
+    foo: i32,
     #[sea_orm(from_col = "bar2")]
-    _bar: String,
+    bar: String,
 }
 
 #[derive(FromQueryResult, DerivePartialModel)]
 #[sea_orm(entity = "Entity")]
 struct FieldFromDiffNameColumnTest {
     #[sea_orm(from_col = "foo2")]
-    _foo: i32,
+    foo: i32,
     #[sea_orm(from_col = "bar2")]
-    _bar: String,
+    bar: String,
 }
 
 #[derive(FromQueryResult, DerivePartialModel)]
 struct FieldFromExpr {
     #[sea_orm(from_expr = "Column::Bar2.sum()")]
-    _foo: f64,
+    foo: f64,
     #[sea_orm(from_expr = "Expr::col(Column::Id).equals(Column::Foo)")]
-    _bar: bool,
+    bar: bool,
+}
+
+#[derive(FromQueryResult, DerivePartialModel)]
+struct Nest {
+    #[sea_orm(nested)]
+    foo: SimpleTest,
+}
+
+#[derive(FromQueryResult, DerivePartialModel)]
+struct NestOption {
+    #[sea_orm(nested)]
+    foo: Option<SimpleTest>,
+}
+
+#[derive(FromQueryResult, DerivePartialModel)]
+#[sea_orm(entity = "bakery::Entity")]
+struct Bakery {
+    id: i32,
+    #[sea_orm(from_col = "Name")]
+    title: String,
+}
```

Comment:
@Goodjooy since you're also a contributor. I realised we've made a small breaking change here, the `from_col` expression changed from `snake_cake` to `TypeCase`, corresponding to the `Column` enum directly. may be this is more consistent from a design sense?

what's your view?

Reply:
I realised it actually still works with lower case